scripts/agent-local/rocks.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This whole file needs to go to: https://github.com/librenms/librenms-agent/pulls